controllers/users-controller.js

```diff
@@ -26,13 +26,32 @@ exports.registerUser = (appReq, appRes) => {
   });
 
   /**
-   * TODO
-   * 1. validate to ensure null values are passed for name,
-   *   email, or password.
-   * 2. verify email is valid i.e. '....'@'...'.'...'
-   * 3. validate for secure passwords and no easy to guess passwords
+   *  Checks whether any of the fields are empty while submission.
+   *  Checks whether the email address is of a valid format
+   *  Checks whether password is of minimum 6 characters & that it has atleast one number & atleast one specail character. 
    */
 
+  if(!appReq.body.email || !appReq.body.name || !appReq.body.password){
```

Comment:
This is behaving the opposite of what you want. If I understand correctly, you want to validate email and password if all user credentials are not null else notify that a field is empty. This isn't doing that - what you have is equivalent to saying if **any** user credential is null proceed to validating email and password.

What you want is a statement equivalent to all must be true to proceed to validating email/password else notify empty field. Let me know if you want me to expand further on this.